### ISSUE TITLE
Windows: nk3 piv application not found

### ISSUE THREAD

**Issue by stnu-iuk** [Nitrokey/pynitrokey#564]
os: Win 11 23H2
firmware: v1.7.2-test.20240813 (latest test version, which - to my understanding - includes the "personal identity validation" functionality)
nitropy-version: 0.4.50; also tested with: 0.4.46
used nitrokeys: Nitrokey 3C NFC and Nitrokey 3A NFC

steps to reproduce:

connect nitrokey to usb port
start nitropy in cmd or powershell with admin rights
used command: nitropy.exe nk3 piv 
alternative used command: nitropy.exe nk3 piv init

expected behavior
piv app is launched, options appear, piv functionality is activated

observed behavior: 

> C:\Program Files\Nitrokey\Nitropy>nitropy.exe nk3 piv init
> Command line tool to interact with Nitrokey devices 0.4.46
> Usage: nitropy.exe nk3 [OPTIONS] COMMAND [ARGS]...
> Try 'nitropy.exe nk3 --help' for help.
> 
> Error: No such command 'piv'.
> 

thus i cannot use the piv functionality, even if not yet stable, which would probably very useful to us.

**Comment by robin-nitrokey** [comment]
As the PIV support is still experimental, it is not available in the released pynitrokey versions yet.  It is added in this pull request: https://github.com/Nitrokey/pynitrokey/pull/506

To test it, you need to install pynitrokey from the Git branch of that pull request, for example:

```
pipx install git+https://github.com/Nitrokey/pynitrokey.git@piv-rebase
```

**Comment by robin-nitrokey** [comment]
The `piv` command is available as an experimental feature since [v0.6.0](https://github.com/Nitrokey/pynitrokey/releases/tag/v0.6.0) so it is no longer necessary to install pynitrokey from Git.
